django_material_demo/Makefile

```diff
@@ -0,0 +1,41 @@
+# Put GENERATE_SECRET into files you want to generate secret
+# Usage: `make generate-secret FROM=/path/to/src TO=/path/to/dest`
+.PHONY: generate-secret
+generate-secret:
+	curl -s https://raw.githubusercontent.com/oursky/devsecops-secret/v1.0/generate-secret.sh \
+		| bash -s -- ${FROM} ${TO}
+
+.PHONY: secret
+secret:
+	make generate-secret FROM=.env.example TO=.env
+
+.PHONY: setup
+setup: secret
+	docker compose up --build --no-start
+	make migrations
+	make migrate
+	make load-data
+	
+.PHONY: start
+start:
+	docker compose up
+
+.PHONY: clean
+clean:
+	docker compose down
+
+.PHONY: migrations
+migrations:
+	docker-compose run --rm web python manage.py makemigrations
+
+.PHONY: migrate
+migrate:
+	docker-compose run --rm web python manage.py migrate
+
+.PHONY: load-data
+load-data:
+	docker-compose run --rm web python manage.py loaddata polls/sample_data.json
+
+.PHONY: create-superuser
+create-superuser:
+	docker-compose run --rm web python manage.py createsuperuser
```

Comment:
https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

`docker compose` and `docker-compose` are not the same, though they are mostly compatible. There is no reason to mix them in the same project. Please choose to keep either one of them.